### PR TITLE
gluon-mesh-b*: name interface name in respondd statistics

### PR DESCRIPTION
**Pull request by lemoer**
This is a prepatory commit to allow showing the airtime statistics and the wifi frequency on the status page.

### CODE REVIEW

**Comment by NeoRaider** [comment]
The interface name is an implementation detail; we usually identify interfaces by their BSSIDs in respondd data.

In addition, changes to these respondd providers should be based on #1879.

**Comment by lemoer** [comment]
Is there a mapping between BSSID and interface name available in the status-page frontend?

**Comment by NeoRaider** [comment]
I'm not sure, but if it isn't, we can add it.

**Comment by christf** [comment]
> The interface name is an implementation detail; we usually identify interfaces by their BSSIDs in respondd data.
> 
> In addition, changes to these respondd providers should be based on #1879.

It should be based on #1799. The idea was to merge the refactoring then the load fix then everything else.

**Comment by lemoer** [comment]
I updated the code to use bssid. A rebase is not done yet.

**Comment by lemoer** [comment]
As neoraiders #1879 is now merged, should I wait till christfs #1799 is also merged?

File: package/gluon-mesh-babel/src/respondd.c [hunk 1]
```diff
@@ -620,6 +620,7 @@ static struct json_object * get_wifi_neighbours(const char *ifname) {
 		json_object_object_add(obj, "signal", json_object_new_int(entry->signal));
 		json_object_object_add(obj, "noise", json_object_new_int(entry->noise));
 		json_object_object_add(obj, "inactive", json_object_new_int(entry->inactive));
+		json_object_object_add(obj, "iface", json_object_new_string(ifname));
```
Comment (NeoRaider) [inline package/gluon-mesh-babel/src/respondd.c]: For Babel, the link-local IP address should be used.

**Comment by NeoRaider** [comment]
> As neoraiders #1879 is now merged, should I wait till christfs #1799 is also merged?

I don't think so - #1799 is currently blocked on a major libbabelhelper rework (https://github.com/christf/libbabelhelper/pull/9) and will require significant changes afterwards. Feel free to get this PR in shape, and we can get it in first.

**Comment by rotanid** [comment]
@lemoer ping, seems like something small we get still get into v2020.2

**Comment by lemoer** [comment]
@rotanid I currently do not have a babel site config to test this with babel. That's the reason why this is stuck.

**Comment by NeoRaider** [comment]
I have just checked the code and found that this is redundant: The neighbours are already nested in an object that has the interface BSSID as its key. No special handling is necessary for Babel after all.

**Pull request closed without merging**